src/gameClasses/Projectile.js

```diff
@@ -86,17 +83,11 @@ var Projectile = IgeEntityPhysics.extend({
 			ige.server.totalProjectilesCreated++;
 		} else if (ige.isClient) {
 			if (currentState) {
-				var defaultAnimation = this._stats.animations[currentState.animation];
-				this.createPixiTexture(defaultAnimation && defaultAnimation.frames[0] - 1, data.defaultData);
+				/* TEMPORARY */
+				//TODO - add new logic for trackEntityById and updating entities with out PIXI in EntityManager
+				ige.pixi.trackEntityById[this.entityId] = undefined;
```

Comment:
this should probably be set to `this` and without the condition.

Also, I've noticed animations not working anymore (explosion in default game).